pages/agent/v3/unclustered_tokens.md

```diff
@@ -0,0 +1,143 @@
+# Unclustered agent tokens
+
+> 🚧 This page documents a deprecated feature of Buildkite
+> Previously, agents only connected to Buildkite directly via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines across to a [cluster](/docs/clusters/overview).
+> Agent tokens are now associated with clusters, such that these tokens connect to Buildkite through a specific cluster within an organization. Refer to the main [agent token](/docs/agent/v3/tokens) for details on how to create agent tokens for clusters.
+> Any new Buildkite organizations created after the official release of clusters on February 26, 2024 will not have the ability to create unclustered agents. Therefore, unclustered agent tokens will not be relevant to these organizations.
+
+Any Buildkite organizations created before February 26, 2024, have an _Unclustered_ area for managing _unclustered agents_ (accessible through _Agents_ > _Unclustered_ of the Buildkite interface), where an _unclustered agent_ refers to any agent that is not associated with a cluster.
```

Comment:
Have another look at this and the next two paragraphs (through the default token section). These are not super clear yet, but uff that's a lot mouthfuls of clustering and unclustering and organizations to get across!

I thiiiink (from reading this) that part of the challenge to describing this clearly is that people may have some unclustered agents and some "new world" agent tokens at the same time, as they slowly migrate?

Reply:
Thanks heaps for your review of this WIP so far! Great pickups on some typos there :grin:!
I'll have a look at the rest of these on Monday.

Reply:
Great to have a 2nd pair of eyes on this note, looking back at this now I can see how this is a tad confusing and I don't think I had all the details on the exact behaviour back then. Anyway, I will re-write to make this clearer, hopefully! 🤞

Reply:
Just for more info, customers with:

- any existing Buildkite orgs (i.e. created before the 26th of Feb) will have an 'unclustered' area in the 'Agent Clusters' page of the Buildkite interface
- Buildkite orgs created after the 26th of Feb will not have an 'unclustered' area in the 'Agent Clusters' page of the Buildkite interface. In other words, unclustered agents will not work with these newer Buildkite orgs.

Therefore, for existing customers who create Buildkite orgs after the 26th of Feb, it's likely that they may have some Buildkite orgs with an 'unclustered' area and others which don't.

If they only really manage a single Buildkite org, that it will comply with the rules explained above.